spec/models/manageiq/providers/kubernetes/container_manager/refresh_parser_spec.rb

```diff
@@ -3,6 +3,39 @@
 describe ManageIQ::Providers::Kubernetes::ContainerManager::RefreshParser do
   let(:parser)  { described_class.new }
 
+  describe "parse_namespaces" do
+    it "handles simple data" do
+      expect(parser.send(
+               :parse_namespaces,
```

Comment:
indent the method parameters two spaces after the start of the line above like you did everything else